pkgs/development/python-modules/ansible-lint/default.nix

```diff
@@ -18,13 +18,13 @@
 
 buildPythonPackage rec {
   pname = "ansible-lint";
-  version = "5.4.0";
+  version = "6.0.2";
```

Comment:
That one was a bit more involved and was already done in https://github.com/NixOS/nixpkgs/pull/166283